packages/tilp.rb

```diff
@@ -0,0 +1,57 @@
+require 'package'
+
+class Tilp < Package
+  description 'TiLP is a linking program for Texas Instruments\' graphing calculators.'
+  homepage 'http://lpg.ticalc.org/prj_tilp/'
+  version '1.18'
+  source_url 'https://www.ticalc.org/pub/unix/tilp.tar.gz'
+  source_sha256 '6ba834f7fdbbce9818ccaa864222aed2d1688b210e9ff2c59576d1fde5159cd7'
+
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'libarchive'
+  depends_on 'libglade'
+  depends_on 'libusb'
+
+  def self.install
+    system 'wget http://lpg.ticalc.org/prj_tilp/download/install_tilp.sh'
+    abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest( File.read('install_tilp.sh') ) == '6baf4b79100a938c2c36218d50c30a39c2beec17490784e3623d4aeebe9931ee'
+    system "PREFIX=#{CREW_PREFIX} SRCDIR=#{CREW_DEST_PREFIX}/share bash install_tilp.sh"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
```

Comment:
I have already tried using `#{CREW_DEST_PREFIX}` and it doesn't work as expected.  Rather than rewrite the install script, in order to track all the files, I needed this crazy copy logic.  I figured the tilp directory was a working directory and not needed.

Reply:
It's annoying that #{CREW_DEST_PREFIX} doesn't do the job... this series of mkdir -p and cp / cp -r scares me :)
Do you have to go through such pain with other autotools-based programs ? There are still many of those.

Reply:
It would normally work except that the dependencies are needed through the progression of the build and if they don't install in #{CREW_PREFIX} first, the build fails.  It might make sense to break out all the dependencies into separate packages.  This is probably more trouble than it's worth, however.  Just so you know, the mkdir -p / cp / cp -r stuff is only there to track the meta data that can be used to remove the files when the package is uninstalled.

Reply:
Using separate packages for libticonv, libtifiles2, libticables2, libticalcs2, gfm (less useful) and tilp is indeed what all Linux distros which package these, i.e. most of the major (non-enterprise) ones, do. One package per autotools definition is the way to go :)